draft-ietf-quic-transport.md

```diff
@@ -6585,22 +6596,25 @@ encrypted with keys derived from the TLS key exchange.  Further, parameter
 negotiation is folded into the TLS transcript and thus provides the same
 integrity guarantees as ordinary TLS negotiation.  An attacker can observe
 the client's transport parameters (as long as it knows the version-specific
-salt) but cannot observe the server's transport parameters and cannot influence
-parameter negotiation.
+salt) but cannot observe the server's transport parameters as sent to
+the client and cannot influence
+parameter negotiation.  (The attacker could make its own connection to
+the server and observe the transport parameters sent on that
+connection.)
```

Comment:
I don't think that this adds much value and so I would prefer not to take this change.

Reply:
Okay; reverted.